packages/framework-types/src/config.ts

```diff
@@ -127,7 +127,14 @@ export class BoosterConfig {
   }
 
   public get tokenVerifier(): TokenVerifier | undefined {
-    return this._tokenVerifier
+    if (this._tokenVerifier) return this._tokenVerifier
+    if (process.env[JWT_ENV_VARS.BOOSTER_JWT_ISSUER] && process.env[JWT_ENV_VARS.BOOSTER_JWKS_URI]) {
+      return {
+        issuer: process.env[JWT_ENV_VARS.BOOSTER_JWT_ISSUER] as string,
+        jwksUri: process.env[JWT_ENV_VARS.BOOSTER_JWKS_URI] as string,
+      }
+    }
```

Comment:
I'm not sure if these new env variables will require updating the auth rocket docs but just saying as a reminder, not a real blocker for now 😄